scenarios/happypath/list/list_applications.go

```diff
@@ -31,5 +31,5 @@ func Applications(env env.Env, suiteName string) (bool, error) {
 		}
 	}
 
-	return err == nil && len(showAppOk.Payload) > 0, err
+	return err
```

Comment:
```suggestion
	if len(showAppOk.Payload) == 0 {
        return errors.New("list application returned empty list of applications")
    }

    return err
```
The original test also checked the length of howAppOk.Payload, and only reported successful test if len(howAppOk.Payload) > 0. Should we keep this test?